js/wasm/ethkey/build.sh

```diff
@@ -0,0 +1,16 @@
+# Remove previous build to avoid name conflicts
```

Comment:
It doesn't need to. The built wasm is in the repo, this script isn't being run when building Parity.